docs/content/_docs/query_language.html

```diff
@@ -244,52 +202,175 @@ <h4>Operators</h4>
   </tr>
 
   <tr>
-    <th scope="row">Tag Starts With</th>
-    <td><pre><code class="language-hql">&lt;a&gt; ^ &lt;b&gt;</code></pre></td>
-    <td><pre><code class="language-json">["^", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
+    <th scope="row">Key Matches</th>
+    <td><pre><code class="language-hql">$key = &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["key", &lt;a&gt;]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Key Does Not Match</th>
+    <td><pre><code class="language-hql">$key != &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["key", &lt;a&gt;]]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag True</th>
+    <td><pre><code class="language-hql">[true]</code></pre></td>
```

Comment:
I'm pretty sure the HQL for true/false doesn't have the brackets around it. They also aren't tag related, they're just literal true and false. Titling it as `Tag True` would make me think its checking the truthiness of a tag.